sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +150,159 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun showAnnotations(
+    annotations: List<View>,
+    edgeInsets: EdgeInsets?,
+    bearing: Double?,
+    pitch: Double?,
+    animationDurationMs: Long
+  ) {
+    if (annotations.isNotEmpty()) {
+      val viewAnnotationOptions = annotations.mapNotNull {
+        getViewAnnotationOptionsByView(it)
+      }
+      val coordinateBounds = coordinateBoundsFromAnnotations(viewAnnotationOptions)
+      val paddings = calculateEdgeInset(viewAnnotationOptions, edgeInsets)
+      val geometry = GeometryCollection.fromGeometries(
+        listOf(
+          coordinateBounds.northeast,
+          coordinateBounds.southwest
+        )
+      )
+      // create overview viewport state with geometry, padding and animation duration
+      val overviewViewportState = mapView.viewport.makeOverviewViewportState(
```

Comment:
what of viewport is not on the classpath?

Reply:
Yeah either we can check if its available or simply use `map.flyTo` as discussed [here](https://github.com/mapbox/mapbox-maps-android/pull/1753#discussion_r1001981929)